templates/view/list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "layout.html" %}
 {% set title = _("Drop Point List") %}
 {% block header %}
-    <link href="{{ url_for('static', filename='lib/css/dataTables.bootstrap.min.css') }}" rel="stylesheet">
+    <link href="{{ url_for('static', filename='lib/css/dataTables.bootstrap4.min.css') }}" rel="stylesheet">
```

Comment:
:+1: And I was already wondering why the table was so ugly... how did I miss this?!